src/Mvc/benchmarkapps/BasicApi/Startup.cs

```diff
@@ -125,7 +125,7 @@ public void ConfigureServices(IServiceCollection services)
             services
                 .AddMvcCore()
                 .AddAuthorization()
-                .AddJsonFormatters(json => json.ContractResolver = new CamelCasePropertyNamesContractResolver())
+                .AddNewtonsoftJsonFeatures(options => options.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver())
```

Comment:
Thoughts on the name? It's a bit more than serializers. There's one casualty here - this method will add services used by views. In the past, you could use MvcCoreBuilder based extensions to avoid adding any view related features to DI. Do we need to maintain a `AddNewtonsoftJsonFormatters` in addition to this?

Reply:
What about just `AddNewtonsoftJson`?

I think it's OK that we're adding the viewfeatures-defined services. They don't do anything unless they are used.

One thing we could do is move the interface types into Mvc.Abstractions.

Reply:
That would reduce the number of assemblies that get loaded.

Reply:
`TempDataSerializer` doesn't quite feel like something that belongs in abstractions. Perhaps it's ok to leave this as-is unless we get some strong feedback